platforms/common/uORB/uORBManager.cpp

```diff
@@ -615,7 +622,11 @@ void uORB::Manager::GlobalSemPool::init(void)
 		sem.init();
 	}
 
+#if defined(__PX4_NUTTX)
+	nxmutex_init((mutex_t *)&_semLock);
```

Comment:
should we check return value of this? what happens if we cannot init mutex?

Reply:
It can't fail (the nxsem_init always succeeds and returns OK) but yes, it would be nice to check the return value regardless. This PR only changes the existing call of px4_sem_init, and the return value check was not there previously. The init is called from uORB::Manager::Manager() constructor, so actually this all assumes that this part can't fail. Maybe it should just print an error trace or something, but not adding it now because again, it just can't fail.